deploy/bootstrap/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '2'
+
+services:
+  consensus:
+    image: ${DOCKER_IMAGE}:${DOCKER_TAG}
+    working_dir: /opt/orbs/projects/consensus-service-typescript/
+    command: node dist/index.js /opt/orbs/config/topologies/discovery/node1/consensus.js
```

Comment:
Why is it `node1` everywhere?

Reply:
Because it's a single configuration suitable for any node.

Reply:
Ah, I see. Perhaps we will rename it to `node` in the future.